src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,209 @@
+import { useState, useContext } from "react";
+import { v4 as uuidv4 } from "uuid";
+import TextArea from "../TextArea";
+import Icon from "../Icon";
+import Tag from "../Tag";
+import Button from "../Button";
+import { format } from "date-fns";
+import { supabase } from "../../config/apiClient";
+import { AppContext } from "../../App";
+import classNames from "classnames";
+const Todo = ({ todo }) => {
+  const {
+    handleRemoveTodo,
+    flag,
+    removeCompleteFromIncomplete,
+    toasts,
+    setToasts,
+    search,
+  } = useContext(AppContext);
+  const [newName, setNewName] = useState(todo.name);
+  const [showEdit, setShowEdit] = useState(false);
+  const [showLoading, setShowLoading] = useState(false);
+  let completedDays = Math.ceil(
+    Math.abs(Date.parse(todo.completed_on) - Date.parse(todo.created_at)) /
```

Comment:
do we really need to use Math and these calculations? Can you find a library for this?

Reply:
used date-fns library to eradicate this calculation